daemon/src/maker.rs

```diff
@@ -176,70 +175,52 @@ async fn main() -> Result<()> {
     )
     .await?;
 
+    db::run_migrations(&db)
+        .await
+        .context("Db migrations failed")?;
+
+    // Create actors
+    let mut conn = db.acquire().await?;
+
+    housekeeping::transition_non_continue_cfds_to_setup_failed(&mut conn).await?;
+    housekeeping::rebroadcast_transactions(&mut conn, &wallet).await?;
+
+    let ActorSystem {
+        cfd_actor_addr,
+        cfd_feed_receiver,
+        order_feed_receiver,
+        update_cfd_feed_receiver,
+    } = ActorSystem::new(
```

Comment:
I think we `unwrap` in one of these constructors because we coulnd't return a `Result` before, can we clean that up as well?